xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -797,6 +814,7 @@ static void DisplayReconfigured(CGDirectDisplayID display,
   m_nWidth = newWidth;
   m_nHeight = newHeight;
   m_glContext = context;
+  g_graphicsContext.SetFPS(m_refreshRate);
```

Comment:
@FernetMenta does this test build correct what you spotted?

http://mirrors.kodi.tv/test-builds/osx/x86_64/kodi-20170103-e6d544f-vtb_choppy_bp-x86_64.dmg